.drone.yml

```diff
@@ -47,8 +44,10 @@ pipeline:
       - yum -y install createrepo
       - createrepo --update /pub/metwork/continuous_integration/rpms/${DRONE_BRANCH}/${OS_VERSION}
       - if test "${OS_VERSION}" = "centos6"; then rm -Rf /pub/metwork/continuous_integration/docs/${DRONE_BRANCH}/mfext; mkdir -p /pub/metwork/continuous_integration/docs/${DRONE_BRANCH}/mfext ; cp -Rf html_doc/* /pub/metwork/continuous_integration/docs/${DRONE_BRANCH}/mfext/ ; fi
+      - touch /buildcache/build_mfext_${DRONE_BRANCH}_`cat .drone_cache`
```

Comment:
because the file is created here, you can remove the corresponding line in the "build" step and remove `/buildcache` volume (in the build "step" also).